assets/out

```diff
@@ -131,7 +131,18 @@ elif [ -n "$build" ]; then
     expanded_build_args=${expanded_build_args}" "$(jq -r 'with_entries(.key |= "--build-arg " + . )|with_entries(.key = .key + "=" +.value)|keys|join(" ")' <$build_args_file)
   fi
 
-  docker build -t "${repository}:${tag_name}" $expanded_build_args -f "$dockerfile" "$build"
+  already_built=false
+  if [ "$if_not_exists" = "true" ]; then
```

Comment:
I am not sure what you mean with this. The change as is was working for me on 2.x.x. I did not succeed to make it work on 3.x.x because of some privilege issues. If something changed recently this code does not handle it.